FAQ/FAQ-Deployment.md

```diff
@@ -17,6 +17,7 @@ Here are common questions about installation, regardless of the deployment metho
 * [With the docker deployment, how to use a different volume for the data?](FAQ-Deployment.md#with-the-docker-deployment-how-to-use-a-different-volume-for-the-data)
 * [How do I activate Tor on my BTCPay Server?](FAQ-Deployment.md#how-do-i-activate-tor-on-my-btcpay-server)
 * [Why activate Tor? Does it mean that nobody knows who I am?](FAQ-Deployment.md#why-activate-tor-does-it-mean-that-nobody-knows-who-i-am)
+* [How can I deactivate Tor (or any other option)?](FAQ-Deployment.md#how-can-i-deactivate-tor-or-any-other-option)
```

Comment:
`or any other option` is unclear, nobody will search for how to deactivate any other option. I think you should re-write this to be plugin neutral and provide Tor as an example.

Reply:
I think:
How to deactivate environment variable in BTCPay?

Reply:
Mmmhhh not sure if "environment variable" will be very clear for the average user. Do we have a complete list of all the possible options somewhere?

Reply:
otoh people that messing with the command line like this probably know about it

Reply:
yes here is the complete list: https://github.com/btcpayserver/btcpayserver-docker#environment-variables

Reply:
Ok cool, seems like we could add `opt-add-tor` to this list. Thanks! I'll get back to this later

Reply:
No, opt-add-tor is below under the BTCPAYGEN_ADDITIONAL_FRAGMENTS list. Might be a good idea to link the "Warning" to your "Why activate Tor?" though :)